stocks/models.py

```diff
@@ -275,8 +274,6 @@ def value_on(self, date=None):
         Returns the current value of the stock configuration
         """
         value = self.stock.latest_quote(date).value * self.quantity
-        if math.isnan(value):
```

Comment:
This code was unreachable (self.quantity cannot be null and must be a float, so if value could be nan it would have already thrown an exception)